server/src/test/java/org/apache/druid/metadata/IndexerSQLMetadataStorageCoordinatorTest.java

```diff
@@ -1268,13 +1283,24 @@ public void testRetrieveUnusedSegmentsUsingNoIntervalsNoLimitAndNoLastSegmentId(
     );
     Assert.assertEquals(segments.size(), actualUnusedSegments.size());
     Assert.assertTrue(segments.containsAll(actualUnusedSegments));
+
+    final ImmutableList<DataSegmentPlus> actualUnusedSegmentDtos = retrieveUnusedSegmentDtos(
```

Comment:
Rename all the old references in this file from `retrieveUnusedSegmentDtos` -> `retrieveUnusedSegmentsPlus`?

Reply:
fixed